src/include/network/routines.rb

```diff
@@ -722,44 +722,41 @@ def GetAllInterfaces
       end
     end
 
-    # Calls wicked ifup if it is the currenct backend, set the link up
-    # otherwise
+    # Wrapper to call 'ip link set up' with the given interface
+    #
+    # @param [String] name of interface to 'set link up'
+    def SetLinkUp(dev_name)
+      log.info("Setting link up for interface #{dev_name}")
+      Run("ip link set #{dev_name} up")
+    end
+
+    # Wrapper to call 'ip link set down' with the given interface
+    #
+    # @param [String] name of interface to 'set link down'
+    def SetLinkDown(dev_name)
+      log.info("Setting link down for interface #{dev_name}")
+      Run("ip link set #{dev_name} down")
+    end
+
+    # Calls wicked ifup with the given interface
     #
     # @param [String] name of interface to put down
     def SetIfaceUp(dev_name)
       log.info("Setting interface #{dev_name} up")
-      Run("ifup #{dev_name} up")
+      Run("ifup #{dev_name}")
     end
 
-    # Calls wicked ifdown if it is the current backend, set the link down
-    # otherwise
+    # Calls wicked ifdown with the given interface
     #
     # @param [String] name of interface to put down
     def SetIfaceDown(dev_name)
       log.info("Setting interface #{dev_name} down")
       Run("ifdown #{dev_name}")
     end
 
-    # Tries to set all interfaces up
-    #
-    # @return [boolean] false if some of interfaces cannot be set up
-    def SetAllIfacesUp
-      interfaces = GetAllInterfaces()
-
-      return false if interfaces.empty?
-
-      interfaces.all? { |i| SetIfaceUp(i) }
-    end
-
-    # Tries to set all interfaces down
-    #
-    # @return [boolean] false if some of interfaces cannot be set down
-    def SetAllIfacesDown
-      interfaces = GetAllInterfaces()
-
-      return false if interfaces.empty?
-
-      interfaces.all? { |i| SetIfaceDown(i) }
+    # Tries to set the link up of all available interfaces
+    def SetAllLinksUp
```

Comment:
This method will be also removed in #453 when the only client that is using it be removed.